zboss/include/addons/zcl/zb_zcl_thermostat_ui_addons.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef ZB_ZCL_THERMOSTAT_UI_ADDONS_H__
+#define ZB_ZCL_THERMOSTAT_UI_ADDONS_H__
+
+#include "zboss_api.h"
+
+/*! \addtogroup zb_zcl_thermostat_ui_addons */
+/*! @{ */
+
+/**@brief Thermostat UI cluster attributes according to ZCL Spec 6.6.2.2 */
+
+typedef struct {
+    uint8_t temperature_display_mode;
+    uint8_t keypad_lockout;
+    uint8_t schedule_programming_visibility;
```

Comment:
The `ScheduleProgrammingVisibility` attribute is optional, thus it is not required by the `ZB_ZCL_DECLARE_THERMOSTAT_UI_CONFIG_ATTRIB_LIST` macro.
Generally, the full list of attributes should have `attrs_ext_t` suffix and be passed to the macro with `ATTRIB_LIST_EXT` suffix.

Do you use the `schedule_programming_visibility` field? If so, could you provide macro definitions for the extended variant?
- `ZB_ZCL_DECLARE_THERMOSTAT_UI_CONFIG_ATTRIB_LIST_EXT`
- `ZB_SET_ATTR_DESCR_WITH_ZB_ZCL_ATTR_THERMOSTAT_UI_CONFIG_SCHEDULE_PROGRAMMING_VISIBILITY_ID`

If not, please simply remove this field or rename the type definition :slightly_smiling_face: